src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx

```diff
@@ -161,7 +161,7 @@ export default function DelegatedSupportCampsUI({
               Are you sure, you want to remove your delegate support given to
               pranav_telentelgia under the topic{" "}
```

Comment:
Is it a typo?

Reply:
This typo is added by vikas, I just have prettified this file